wgpu-precompile-macro/src/lib.rs

```diff
@@ -193,6 +193,15 @@ impl Parse for PrecompileDxilArgs {
     }
 }
 
+struct TempFolder {
+    path: PathBuf,
+}
+impl Drop for TempFolder {
+    fn drop(&mut self) {
+        std::fs::remove_dir_all(&self.path).expect("Failed to remove temporary folder");
```

Comment:
```suggestion
        let _ = std::fs::remove_dir_all(&self.path);
```

We can't panic if we're already unwinding, lets do our best to clean up, but if that fails, we can just move on. Panicing while unwinding will double-fault and abort the program.